docs/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.md

```diff
@@ -23,7 +23,7 @@ Provides access to the Outlook add-in object model for Microsoft Outlook.
 
 | Property | Minimum<br>permission level | Modes | Return type | Minimum<br>requirement set |
 |---|---|---|---|:---:|
-| [diagnostics](/javascript/api/outlook/office.mailbox?view=outlook-js-preview&preserve-view=true#outlook-office-mailbox-diagnostics-member) | ReadItem | Compose<br>Read | [Diagnostics](/javascript/api/outlook/office.diagnostics?view=outlook-js-preview&preserve-view=true) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [diagnostics](/javascript/api/outlook/office.mailbox?view=outlook-js-preview&preserve-view=true#outlook-office-mailbox-diagnostics-member)<br><br>**Note**: Starting with requirement set 1.5, you can also use the [Office.context.diagnostics](office.context.md#diagnostics-contextinformation) property to get similar information | ReadItem | Compose<br>Read | [Diagnostics](/javascript/api/outlook/office.diagnostics?view=outlook-js-preview&preserve-view=true) | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```

Comment:
I don't think you need to add this note to the properties table. The person will get this and other info when they follow the link. Here and other reqt. set versions.

Reply:
That's a good point. I've removed the note from the properties table.